### PR TITLE
remove the word hack

### PR DESCRIPTION
**Pull request by zsd4yr**
#143

### CODE REVIEW

File: src/System.Windows.Forms/src/System/Resources/ResXResourceWriter.cs [hunk 1]
```diff
@@ -297,11 +297,11 @@ mimetype set.
             if (xmlTextWriter == null) {
                 // 
 
-                bool writeHeaderHack = false;
+                bool writeHeaderHeaderRequired = false;
```
Comment (OliaG) [inline src/System.Windows.Forms/src/System/Resources/ResXResourceWriter.cs]:
```suggestion
                bool writeHeaderRequired = false;
```

File: src/System.Windows.Forms/src/System/Windows/Forms/VisualStyles/VisualStyleRenderer.cs [hunk 2]
```diff
@@ -1032,7 +1032,9 @@ public Font GetFont(IDeviceContext dc, FontProperty prop)
                 // Let all threads know their cached handles are no longer valid; 
                 // cache refresh will happen at next handle access.
                 // Note that if the theme changes 2^sizeof(long) times before a thread uses 
-                // its handle, this whole version hack won't work, but I don't see that happening.
+                // its handle, this whole version check won't work, but I don't see that happening.
```
Comment (OliaG) [inline src/System.Windows.Forms/src/System/Windows/Forms/VisualStyles/VisualStyleRenderer.cs]:
```suggestion
                // its handle, this whole version check won't work, but it is unlikely to happen.
```

File: src/System.Windows.Forms/src/System/Resources/ResXResourceWriter.cs [hunk 3]
```diff
@@ -297,11 +297,11 @@ mimetype set.
...
 
                 if (textWriter != null) {
                     textWriter.WriteLine("<?xml version=\"1.0\" encoding=\"utf-8\"?>");
-                    writeHeaderHack = true;
+                    writeHeaderHeaderRequired = true;
```
Comment (OliaG) [inline src/System.Windows.Forms/src/System/Resources/ResXResourceWriter.cs]:
```suggestion
                    writeHeaderRequired = true;
```

File: src/System.Windows.Forms/src/System/Resources/ResXResourceWriter.cs [hunk 4]
```diff
@@ -315,7 +315,7 @@ mimetype set.
                 xmlTextWriter.Formatting = Formatting.Indented;
                 xmlTextWriter.Indentation = 2;
 
-                if (!writeHeaderHack) {
+                if (!writeHeaderHeaderRequired) {
```
Comment (OliaG) [inline src/System.Windows.Forms/src/System/Resources/ResXResourceWriter.cs]:
```suggestion
                if (!writeHeaderRequired) {
```

File: src/System.Windows.Forms/src/System/Windows/Forms/Form.cs [hunk 5]
```diff
@@ -5157,10 +5157,10 @@ private void CallShownEvent()
         ///     scale window size non-lineary. If this message is not processed, the size is scaled linearly by Windows.
         ///     This message is sent to top level windows before WM_DPICHANGED.
         ///     If the application responds to this message, the resulting size will be the candidate rectangle 
-        ///     sent to WM_DPICHANGED. The WPARAM contains a DPI value.� The size needs to be computed if 
-        ///     the window were to switch to this DPI.�LPARAM is unused and will be zero. 
+        ///     sent to WM_DPICHANGED. The WPARAM contains a DPI value.� The size needs to be computed if
```
Comment (GSPP) [inline src/System.Windows.Forms/src/System/Windows/Forms/Form.cs]: Something broke here.

**Pull request merged**